app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -1656,6 +1715,15 @@ class ChatActivity :
                                 message.voiceMessageSeekbarProgress = 0
                                 adapter?.update(message)
                                 stopMediaPlayer(message)
+                                if(hasConsecutiveVoiceMessage){
+                                    val defaultMediaPlayer = MediaPlayer.create(context, R.raw
+                                        .next_voice_message_doodle)
```

Comment:
how is the ogg file created/ where is it from? The license must be valid to include it.
If the license is okay, just include it in REUSE.toml file (you will see how it's done for other ogg files there)

Reply:
I made it by myself, I will take a look on how to license it